include/model/ModelTypes.h

```diff
@@ -280,30 +280,6 @@ class MODEL_EXPORT CResultType {
 //!   -# PopulationHighSumByBucketPersonAndAttribute: is for detecting
 //!      unusually high sums of the metric values in a bucketing interval
 //!      for each (person, attribute) pair analyzed as a population.
-//!   -# PeersAttributeTotalCountByPerson: is the count of events for each
-//!      attribute for each person over all time analyzed as categorical
-//!      data.
-//!   -# PeersCountByBucketPersonAndAttribute: is the non-zero count of
-//!      events for each (person, attribute) pair in a bucketing interval
-//!      analyzed by peer group.
-//!   -# PeersUniqueCountByBucketPersonAndAttribute: is the count of unique
-//!      values for each person and attribute in a bucketing interval analyzed
-//!      by peer group.
-//!   -# PeersLowCountsByBucketPersonAndAttribute: is for detecting unusually
-//!      low non-zero counts of events for each (person, attribute) pair in a
-//!      bucketing interval analyzed as a population.
-//!   -# PeersHighCountsByBucketPersonAndAttribute: is for detecting unusually
-//!      high non-zero counts of events for each (person, attribute) pair in a
-//!      bucketing interval analyzed as a population.
-//!   -# PopulationInfoContentByBucketAndPerson: is for detecting unusual
```

Comment:
The last three here aren't starting with `Peers`. Could you double-check we should remove them?

Reply:
This was a typo in the original comment. We already have docs for these above.